.eslintrc.js

```diff
@@ -79,6 +79,9 @@ module.exports = {
     'one-var': 'off',
     'no-constant-condition': ['error', { checkLoops: false }],
 
+    'vue/multi-word-component-names': 'off',
+    'vue/v-on-event-hyphenation': 'off',
```

Comment:
Any reason to do this? Code should be valid under these rules right now.

Reply:
No, it throws errors.

```
/Users/tobiasruck/neko/stamp/src/components/Profile.vue
  1:1  error  Component name "Profile" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/components/Settings.vue
  1:1  error  Component name "Settings" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/components/chat/messages/ChatMessage.vue
  32:7  warning  v-on event '@txClick' must be hyphenated      vue/v-on-event-hyphenation
  33:7  warning  v-on event '@deleteClick' must be hyphenated  vue/v-on-event-hyphenation
  34:7  warning  v-on event '@replyClick' must be hyphenated   vue/v-on-event-hyphenation

/Users/tobiasruck/neko/stamp/src/components/chat/messages/ChatMessageStack.vue
  25:11  warning  v-on event '@replyClicked' must be hyphenated  vue/v-on-event-hyphenation

/Users/tobiasruck/neko/stamp/src/layouts/ChatLayout.vue
  55:11  warning  v-on event '@sendFileClicked' must be hyphenated   vue/v-on-event-hyphenation
  56:11  warning  v-on event '@giveLotusClicked' must be hyphenated  vue/v-on-event-hyphenation

/Users/tobiasruck/neko/stamp/src/layouts/MainLayout.vue
  32:11  warning  v-on event '@toggleContactDrawerOpen' must be hyphenated  vue/v-on-event-hyphenation
  33:11  warning  v-on event '@toggleMyDrawerOpen' must be hyphenated       vue/v-on-event-hyphenation
  34:11  warning  v-on event '@setupCompleted' must be hyphenated           vue/v-on-event-hyphenation

/Users/tobiasruck/neko/stamp/src/pages/Changelog.vue
  1:1  error  Component name "Changelog" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Chat.vue
   1:1  error    Component name "Chat" should always be multi-word  vue/multi-word-component-names
  17:9  warning  v-on event '@replyClicked' must be hyphenated      vue/v-on-event-hyphenation
  67:7  warning  v-on event '@sendFileClicked' must be hyphenated   vue/v-on-event-hyphenation
  68:7  warning  v-on event '@giveLotusClicked' must be hyphenated  vue/v-on-event-hyphenation
  72:7  warning  v-on event '@sendMessage' must be hyphenated       vue/v-on-event-hyphenation

/Users/tobiasruck/neko/stamp/src/pages/Forum.vue
  1:1  error  Component name "Forum" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Profile.vue
  1:1  error  Component name "Profile" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Receive.vue
  1:1  error  Component name "Receive" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Send.vue
  1:1  error  Component name "Send" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Settings.vue
  1:1  error  Component name "Settings" should always be multi-word  vue/multi-word-component-names

/Users/tobiasruck/neko/stamp/src/pages/Setup.vue
  1:1  error  Component name "Setup" should always be multi-word  vue/multi-word-component-names

```